react-components/src/components/CacheProvider/FdmNodeCache.ts

```diff
@@ -228,16 +224,16 @@ export class FdmNodeCache {
   private async getAndCacheRevisionToEdgesMap(
     modelRevisionIds: ModelRevisionId[],
     fetchViews: boolean
-  ): Promise<Map<ModelRevisionKey, FdmEdgeWithNode[]>> {
+  ): Promise<Map<ModelRevisionKey, FdmConnectionWithNode[]>> {
     const revisionIds = modelRevisionIds.map((modelRevisionId) => modelRevisionId.revisionId);
-    const edges = await this.getEdgesForRevisions(revisionIds, this._fdmClient);
+    const connections = await this._fdm3dDataProvider.getCadConnectionsForRevisions(revisionIds);
 
-    const edgesWithOptionalViews = fetchViews
-      ? await this.getViewsForEdges(edges)
-      : edges.map((edge) => ({ edge }));
+    const connectionsWithOptionalViews = fetchViews
+      ? await this.getViewsForConnections(connections)
+      : connections.map((connection) => ({ connection }));
 
     const revisionToEdgesMap = await createRevisionToEdgesMap(
```

Comment:
Maybe it is better to change any `RevisionToEdgesMap` wording to something like `RevisionToConnectionsMap` as well